### PR TITLE
Bump to version 2.18.3

### PR DESCRIPTION
**Pull request by randomorder**

### CODE REVIEW

File: download.sh [hunk 1]
```diff
@@ -12,8 +12,7 @@ GEOSERVER_VERSION=$1
 TEMP_DOWNLOADED=$2 
 
 echo "GeoServer Data Directory is going to be downloaded"
-# for debugging
-echo "curl -k -L -O https://www.dropbox.com/s/q0qc2t7d9alo9fk/data-${GEOSERVER_VERSION}.zip"
-curl -k -L -O https://www.dropbox.com/s/q0qc2t7d9alo9fk/data-$GEOSERVER_VERSION.zip && \
-unzip -x -d ${TEMP_DOWNLOADED} data-$GEOSERVER_VERSION.zip
+artifact_url="https://www.dropbox.com/s/q0qc2t7d9alo9fk/data-$GEOSERVER_VERSION.zip"
```
Comment (giohappy) [inline download.sh]: @randomorder I suspect the dropbox link must be updated here too

Comment (randomorder) [reply download.sh]: Yes, PR updated
thanks!

**Review by afabiani — CHANGES_REQUESTED** [review]
@randomorder since this is back online https://build.geo-solutions.it/geonode/geoserver/latest/
it would be better to point to this base url in order to make this generic.

**Comment by giohappy** [comment]
> @randomorder since this is back online https://build.geo-solutions.it/geonode/geoserver/latest/
it would be better to point to this base url in order to make this generic.

How is this Jenkins reliable and able to cope with savage traffic?
We were considering to use AWS S3 instead, but I'm open to any solution.

**Pull request merged**